compiler+runtime/src/cpp/jank/codegen/llvm_processor.cpp

```diff
@@ -298,14 +306,35 @@ namespace jank::codegen
   /* NOLINTNEXTLINE(readability-make-member-function-const): Can't be const, due to overload resolution ambiguities. */
   llvm::Value *llvm_processor::gen(expr::var_deref_ref const expr, expr::function_arity const &)
   {
-    auto const ref(gen_var(make_box<obj::symbol>(expr->var->n, expr->var->name)));
-    auto const fn_type(
-      llvm::FunctionType::get(ctx->builder->getPtrTy(), { ctx->builder->getPtrTy() }, false));
-    auto const fn(ctx->module->getOrInsertFunction("jank_deref", fn_type));
-
-    llvm::SmallVector<llvm::Value *, 1> const args{ ref };
-    auto const call(ctx->builder->CreateCall(fn, args));
+    llvm::Value *call{};
+    auto const var_qualified_name(make_box<obj::symbol>(expr->var->n, expr->var->name));
+    /*
```

Comment:
Please add a new line before this. Block comments like this should always have one. You'll find that to be true throughout the jank code base.